README.md

```diff
@@ -181,6 +194,25 @@ exporters:
     endpoint: "http://127.0.0.1:9411/api/v2/spans"
 ```
 
+### <a name="config-pipelines"></a>Pipelines
+Pipelines can be of two types:
+
+- metrics: collects and processes metrics data.
+- traces: collects and processes trace data.
+
+A pipeline consists of a set of receivers, processors, and exporters. Each receiver/processor/exporter must be specified
+in the configuration to be included in a pipeline and each receiver/processor/exporter can be used in more than one pipeline.
```

Comment:
There is a slight nuance with processors that is worth mentioning explicitly.

When the same processor is referenced in multiple pipelines each pipelines gets a _separate_ instance of that processor (each instance having the same config but separate internal state). By contrast receivers and exporters referenced from multiple pipelines are the same instance of the receiver and exporter.

So, for example if I have a "queue" processor referenced from 2 pipelines, it really means 2 separate queues, one for each pipeline.

While if I reference "OpenCensus" receiver from 2 pipeline, it is really the same receiver and the data it receives is duplicated and fed into 2 pipelines.

Reply:
That is a good distinction that should be added. I will add it in this pr.  
As the description said, this document change is only a starting point and I'm sure there is much missing from it. Once we have a good skeleton in place (probably needs another pr or two to get it solid), we can fill in remaining gaps.